src/types.ts

```diff
@@ -1,3 +1,6 @@
+import {Octokit} from '@octokit/action'
+export declare type ActionOctokit = InstanceType<typeof Octokit>
```

Comment:
@gr2m I've realized when moving from the `rest` sdk to the `action` sdk that it doesn't work on the exported type straight away since this are the exported types in one and the other:

On rest:
```
import { Octokit as Core } from "@octokit/core";
export { RestEndpointMethodTypes } from "@octokit/plugin-rest-endpoint-methods";
export declare const Octokit: typeof Core & import("@octokit/core/dist-types/types").Constructor<{
    paginate: import("@octokit/plugin-paginate-rest").PaginateInterface;
} & import("@octokit/plugin-rest-endpoint-methods/dist-types/generated/method-types").RestEndpointMethods & import("@octokit/plugin-rest-endpoint-methods/dist-types/types").Api>;
export declare type ActionOctokit = InstanceType<typeof Octokit>
```

On action:
```
import { Octokit as Core } from "@octokit/core";
export { RestEndpointMethodTypes } from "@octokit/plugin-rest-endpoint-methods";
export declare const Octokit: typeof Core & import("@octokit/core/dist-types/types").Constructor<{
    paginate: import("@octokit/plugin-paginate-rest").PaginateInterface;
} & import("@octokit/plugin-rest-endpoint-methods/dist-types/generated/method-types").RestEndpointMethods & import("@octokit/plugin-rest-endpoint-methods/dist-types/types").Api>;
```

As you can see there is an additional line that makes the the type be correctly exported from one or the other. Not sure the reasoning behind this as my ts knowledge is not too deep, but I found that if I create that line and export it with the name the types are correctly detected and compiling. I will open an issue with this info in the action package 😸

Reply:
it's fixed via `@octokit/action@3.15.4`

Reply:
Awesome! Thanks!